c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -1611,7 +1620,7 @@ void MfxC2DecoderComponent::DoWork(std::unique_ptr<C2Work>&& work)
                 if (C2_OK != res) break;
 
                 mfx_sts = DecodeFrame(bs, std::move(frame_out), &flushing, &expect_output);
-            } while (mfx_sts == MFX_ERR_NONE || mfx_sts == MFX_ERR_MORE_SURFACE);
+            } while (mfx_sts == MFX_ERR_MORE_SURFACE);
```

Comment:
Please do not change there as sometimes the input bitstream includes more than one access unit.

Reply:
> Please do not change there as sometimes the input bitstream includes more than one access unit.

if input bs includes more than one access unit, oneVPL will return MFX_ERR_NONE?

Reply:
Yes. This logic here is to process as many more frames as possible.

Reply:
> Yes. This logic here is to process as many more frames as possible.

I think it would be more reasonable for it to return MFX_ERR_MORE_SURFACE, but it's fine, I will revert this change.